openmdao/docs/feature_reference/visualization_and_analysis/output/index.rst

```diff
@@ -1,8 +1,14 @@
-******
-Output
-******
+***********
+Debugging
+***********
+
+
 
 .. toctree::
     :maxdepth: 1
 
-    listing_variables.rst
\ No newline at end of file
+    listing_variables.rst
```

Comment:
Note that there is an outstanding PR for this doc on which you are a requested reviewer.  That story stated that "the index structure is also weird. Too many levels deep.", which resulted in the `output` directory being removed.  It looks like that is no longer requested?